<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matriz</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>NÚMERO DE DESLOCAMENTOS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="_tamanhoMatriz"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>PARÂMETRO &quot;a&quot;:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="_parametroA"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnGerarMatriz">
            <property name="text">
             <string>GERAR MATRIZ</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="btnCalcularMatriz">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>CALCULAR SISTEMA</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnCalcularJordan">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>GAUSS-JORDAN</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRelatorio">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>MOSTRAR RELATÓRIO &gt;</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTableWidget" name="tblCoeficientes"/>
        </item>
        <item>
         <widget class="QTableWidget" name="tblResultado">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>87</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>DESLOCAMENTOS:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTableWidget" name="tblRaizes">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>55</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>AMPLITUDES:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QTableWidget" name="tblAmplitudes">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>55</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="5">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPlainTextEdit" name="pteRelatorio">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMatriz">
    <property name="title">
     <string>Matriz</string>
    </property>
   </widget>
   <addaction name="menuMatriz"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
